docs/solar/README.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Solar API Overview
+---
+
+# Solar API Endpoints
+
+The base URI for all site-specific solar APIs is `https://owner-api.teslamotors.com/api/1/energy_sites/{site_id}/` followed by the endpoint name and parameters. `site_id` identifies the solar installation.  See the [products.md](products.md) file for obtaining the `site_id` value.
+
+The solar endpoints fall into three groups:
+
+### Solar energy generation data endpoints
+
+| Cmd | Endpoint            | Description                                    |
+| :-- | :------------------ | :--------------------------------------------- |
+| GET | `history`           | Current and immediately prior generation data. |
+| GET | `calendar_history`  | Generation data from a past period.            |
+| GET | `telemetry_history` | No information available.                      |
```

Comment:
We don't need to put these endpoints as a table of contents list. You can use Gitbook to link to them. 

To change the main sidebar, you use the Summary file: https://github.com/timdorr/tesla-api/blob/master/docs/SUMMARY.md